README.md

```diff
@@ -192,11 +192,11 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [crystal-mysql](https://github.com/crystal-lang/crystal-mysql) - MySQL connector for Crystal
  * [crystal-pg](https://github.com/will/crystal-pg) - A Postgres driver
  * [crystal-redis](https://github.com/stefanwille/crystal-redis) - Full featured Redis client
+ * [crystal-rethinkdb](https://github.com/kingsleyh/crystal-rethinkdb) - driver for rethinkdb / rebirthdb
```

Comment:
I think the actual brands are RethinkDB and RebirthDB. I know it's a bit nitpick, but that's how it was before. :+1: 

```suggestion
 * [crystal-rethinkdb](https://github.com/kingsleyh/crystal-rethinkdb) - Driver for RethinkDB / RebirthDB
```